### DISCUSSION TITLE
Select all, exclude all.

### DISCUSSION THREAD

**Discussion by smartlegionlab** [codingjoe/django-select2#121] [Q&A]
Hello. There are a lot of elements (300+). Need multiselect, search, select all, exclude all.

Multiselect and searches are fine, but is there an option to select all and exclude all?

**Comment by codingjoe** [comment]
Hi @smartlegionlab,

Excellent question! I don't think there is a solution within Select2, nor do we provide one in this package currently.
The reason is simple, if you're to select all, this would mean you'd need to render all options, which would either cause your server or browser to come to its knees.

Exclusion is an interesting concept. I haven't built anything like that, so I have no examples to share, but it should work with some customizations.

I am curious what solution you'll come up with.

Best Joe

**Reply by smartlegionlab** [reply]
Hello, i used: django_filter + multiselect(JS).
